### ISSUE TITLE
How to init GhostModule

### ISSUE THREAD

**Issue by KopiSoftware** [iamhankai/ghostnet.pytorch#16]
I try to re-implement the Ghost-Resnet56 in the paper, but I can't have the same performance as the author's work (accuracy is about 91% and map is 89.5%). I think the problem is that lacking of the ghost module initialization, but I'm not for sure.
I reviewed the ghost module and nn.Conv2d and I found that ghost module doesn't have a weight initialization operation while ghostnet does. And I found that the initialization doesn't include linear and relu layers.
I wonder how to design a weight initialize function for a resnet56.

```python
    def _initialize_weights(self):
        for m in self.modules():
            if isinstance(m, nn.Conv2d):
                nn.init.kaiming_normal_(m.weight, mode='fan_out', nonlinearity='relu')
            elif isinstance(m, nn.BatchNorm2d):
                m.weight.data.fill_(1)
                m.bias.data.zero_()
```

**Comment by iamhankai** [comment]
Have you modified the 1st conv layer to Ghost module? We did not touch the 1st conv in our experiments.

**Comment by KopiSoftware** [comment]
@iamhankai 
Yep, I do actually replaced all the conv layer in resnet56 just as the paper described in chapter 4.1.2 . I'll give it a try to fix the first layer and replay later.
